Cargo.toml

```diff
@@ -8,3 +8,15 @@ edition = "2018"
 
 [dependencies]
 clap = "^2"
+platform-info = "0.0.1"
+serde = { version = "1.0", features = ["derive"] }
+reqwest = { version = "0.10", features = ["json"] }
+tokio = { version = "0.2", features = ["full"] }
+serde_json = "1.0"
+serde_yaml = "0.8.13"
+uuid = { version = "0.8", features = ["v4"] }
+tiny_http = "0.6"
+url = "2.1.1"
+tungstenite = "0.11.1"
+percent-encoding = "2.1.0"
+ha_api = { git = "https://github.com/kylepotts/ha_api", branch = "kylep-add-as-mobile-app"}
```

Comment:
Would be nice for this to represent the actual upstream once the PR is merged

Reply:
I think that it is find for rapid prototyping. In the future we are going to require that changes be accepted in the library before they merged here

Reply:
My personal preference would be that every commit, CI would generate a snapshot version of the library which the user could point to.

I imagine the process would be as such

1). PR against ha_api repo
2). PR against this repo pointing to snapshot version generated in 1
3). PR in 1 is merged, non snapshot version of the library is created
4) PR in 2 is updated with the non snapshot version

Reply:
I don't love pushing to crates.io on every commit

Reply:
That is fine but we should standardize on some process. I'm fine with pointing to the git repo while were doing active development.

Reply:
I see it going like this. If you are making a change in halcyon that requires changes to api library. you will need to 

1) Open a PR on homeassistant-rs
2) Have PR merged on homeassistent-rs
3) Open PR on halcyon
4) Have PR merged on halcyon

OR

1. Open a PR on homeassistant-rs and hyalcyon (with api pionting at github branch)
2. Have PR merged on homeassistent-rs
3. Change api version to point at the new version pushed when homeassitant-rs PR is merged
4. Have PR merged on halcyon